components/CheckBox.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { Checkbox, Collapse } from "antd";
+import "antd/dist/antd.css";
+
+const { Panel } = Collapse;
+
+function CheckBox({ filters, handleFilters }) {
+  let A: {
+    map: any;
+    id: number;
+    name: string;
+    groupId: number;
+    semester: number;
+  }[] = [];
+  const B = [
+    "1st Year - 1st Semester",
+    "1st Year - 2nd Semester",
+    "2nd Year - 1st Semester",
+    "2nd Year - 2nd Semester",
+    "3rd Year - 1st Semester",
+    "3rd Year - 2nd Semester",
+    "4th Year - 1st Semester",
+    "4st Year - 2nd Semester",
+    "5th Year ",
+    "Other",
+  ];
```

Comment:
I think you should rename this with an useful name.
Naming variables is the worst programmer nightmare, I know 🤣